qp_shotgun/humann2/humann2.py

```diff
@@ -84,50 +85,75 @@ def generate_humann2_analysis_commands(forward_seqs, reverse_seqs, map_file,
             % ', '.join(sn_by_rp.keys()))
```

Comment:
Ok so per our conversation, the 'run prefix' in the mapping file column must be exactly equal to the filename of the forward read. Suggest adding a comment to clarify this behavior.